cmd/krew/cmd/install.go

```diff
@@ -32,7 +31,7 @@ import (
 
 func init() {
 	var forceHEAD *bool
-	var manifest *string
+	var manifest, forceDownloadFile *string
```

Comment:
IMHO `forceXXXXX` does sound like a boolean value. Can we maybe name it `downloadOverwrite`.

Reply:
I'll keep it as is. Feel free to send PR.